javascript/javascript3/week3/ShoppingCart/script.js

```diff
@@ -0,0 +1,87 @@
+// @ts-nocheck
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    // Implement functionality here
+    const urlCurrency = `https://v6.exchangerate-api.com/v6/65e9169f45d75e71ef7414dd/latest/${currency}`;
+    return fetch(urlCurrency)
+      .then((response) => response.json())
+      .then((data) => {
+        const getConvert = data.conversion_rates[currency];
+        //const convert = getConvert * this.price;
+        console.log(getConvert);
+      });
```

Comment:
I think you should get all currency of one unit on the currency. you can use DKK here; we have to log the new prices with the currency rate. Also, you can easily call the function this way without having to console.log it. => `flatscreen.convertToCurrency('USD')`
```suggestion
  convertToCurrency(currency) {
    // Implement functionality here
    const urlCurrency = `https://v6.exchangerate-api.com/v6/65e9169f45d75e71ef7414dd/latest/DKK`;
    return fetch(urlCurrency)
      .then((response) => response.json())
      .then((data) => {
        const convertRate = data.conversion_rates[currency];
        console.log(this.price * convertRate )
      });
```